src/components/tasks/TasksPassword.js

```diff
@@ -0,0 +1,38 @@
+import {useState} from "react";
+import {Button, Card, CardActions, CardContent, TextField, Typography} from "@material-ui/core";
+import {keys, useKeyPress} from "../../utils/hooks/useKeyPress";
+
+export const TasksPassword = ({ handleSubmit }) => {
+
+    const [password, setPassword] = useState("")
+
+    const handleKeyPress = (event) => {
+        // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
To blokuje żeby linter się nie czepiał?

Reply:
Ogólnie w React jest zasada, że kazda funkcja zaczynająca się od use jest hookiem, a hooków możesz używać tylko w komponentach funkcyjnych. Jako że te "hooki" nie korzystają ze standardowych hooków, nie mają takich ograniczeń (ale nazwałem je hookami, bo mają podobne zastosowanie). Linter jest tak skonfigurowany, żeby się o to czepiał, najprościej było dać ten komentarz, a nie chcę grzebać w linterze,  bo albo coś zepsuję, a nawet jeśli nie, to to ustawienie może się jeszcze przydać. 

Tl;dr - tak XDD